### PR TITLE
Create base repository with gulp and jekyll

### PR DESCRIPTION
**Pull request by WillMoggridge**
Created base repository for building website.

- Use `npm run` commands as entry point.
- Gulp has been built with ES20x support with modular imports

`npm run develop` will watch files and start jekyll server
`npm run help` displays other useful options.

### CODE REVIEW

**Comment by WillMoggridge** [comment]
[Create base repository](https://trello.com/c/zouo9u85/2777-create-base-repository)

File: _layouts/base.html [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+
+        <title>{% if page.title %}{{ page.title }}{% else %}{{ site.title }}{% endif %} - conjure-up.io</title>
+
+        <!-- Google Tag Manager -->
+        <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':
+        new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],
+        j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src=
+        'https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);
+        })(window,document,'script','dataLayer','GTM-K92JCQ');</script>
+        <!-- End Google Tag Manager -->
```
Comment (anthonydillon) [inline _layouts/base.html]: Tag manager twice?

File: _layouts/base.html [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+        </script>
+
+        <meta name="description" content="{% if page.excerpt %}{{ page.excerpt | strip_html | strip_newlines | truncate: 160 }}{% else %}{{ site.description }}{% endif %}">
+
+        <!--
+        <link rel="apple-touch-icon-precomposed" sizes="144x144" href="{{ site.asset_path }}3361409d-apple-touch-icon-144x144-precomposed.png" />
+        <link rel="apple-touch-icon-precomposed" sizes="114x114" href="{{ site.asset_path }}5fe4d3c8-apple-touch-icon-114x114-precomposed.png" />
+        <link rel="apple-touch-icon-precomposed" sizes="72x72" href="{{ site.asset_path }}09460d9a-apple-touch-icon-72x72-precomposed.png" />
+        <link rel="apple-touch-icon-precomposed" href="{{ site.asset_path }}cc66da65-apple-touch-icon-precomposed.png" />
+        -->
```
Comment (anthonydillon) [inline _layouts/base.html]: Commented out?

**Comment by grahambancroft** [comment]
Issues fixed in separate branch, this is good to merge 👍

**Pull request merged**